konflux-ci/rbac/core/rbac.yaml

```diff
@@ -178,15 +178,20 @@ rules:
   - verbs:
       - get
       - list
-      - watch
+      - create
```

Comment:
Does create give you watch? I.e. is create enough or we need both?

Reply:
Hm, added `watch` back.